src/index.html

```diff
@@ -6,7 +6,40 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="page__calendar calendar calendar--month-length_29">
```

Comment:
```suggestion
    <div class="calendar calendar--29 calendar--tue">
```
Вот так должны выглядеть классы